src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SnsService.kt

```diff
@@ -73,10 +73,7 @@ class SnsService(hmppsQueueService: HmppsQueueService, private val objectMapper:
           ).also { log.info("Published event $payload to outbound topic") }
       )
     } catch (e: Throwable) {
```

Comment:
Throwable or Exception ?

Reply:
Yeah TBH im not totally sure, we can only trigger these errors in the DEV env by messing with the config - throwable (im hoping) will catch anything major and convert to a friendly exception for the UI / logs.  We could probably drop it completely and let if fail if the config is wrong, but im not sure how quick the cloud team can respond in a prod environment.  Also means we can easily dump the exception into azure for alerts